docs/reference/commandline/system_df.md

```diff
@@ -0,0 +1,64 @@
+<!--[metadata]>
++++
+title = "system df"
+description = "The system df command description and usage"
+keywords = [system, data, usage, disk]
+[menu.main]
+parent = "smn_cli"
++++
+<![end-metadata]-->
+
+# system df
+
+```markdown
+Usage:	docker system df [OPTIONS]
+
+Show docker filesystem usage
+
+Options:
+      --help      Print usage
+  -v, --verbose   Show detailed information of space usage
```

Comment:
Looks like _on_ https://www.usingenglish.com/forum/threads/101778-Difference-bet-Information-on-and-information-about